awxkit/awxkit/api/pages/api.py

```diff
@@ -80,6 +80,9 @@ class Api(base.Base):
 
 
 class ApiV2(base.Base):
+    def __init__(self, connection=None, *a, **kw):
+        super().__init__(connection, *a, **kw)
+        self._has_error = False
```

Comment:
Since this attribute is only used for `import_assets` and `export_assets`, I'd recommend moving setting this there instead of adding this overridden `__init__`

Reply:
in light of way that errors from the ApiV2 level should bubble back up to calling methods in `resource.py` - I believe that we should leave the initialization of `self._has_error` in the constructor

Reply:
done